app/views/output_elements/tables/_merit_order_table.html.haml

```diff
@@ -68,7 +68,9 @@
         'hydro_mountain',
         'hydro_river',
         'local_chp_wood',
-        'local_chp_gas'
+        'local_chp_gas',
+        'waste_chp_ccs',
+        'waste_power_ccs'
```

Comment:
Not sure if this is possible, but it would be nice if all the waste plants would be together in the table:

<img width="608" alt="Screenshot 2021-08-06 at 10 16 55" src="https://user-images.githubusercontent.com/14875123/128480874-becd321d-41e1-4571-bc56-b413de37657b.png">

Reply:
Done with latest commit

![image](https://user-images.githubusercontent.com/33415156/130801854-40b82f78-9635-452a-ac70-8e9e5ec23764.png)